ldpd/ldp_vty_cmds.c

```diff
@@ -0,0 +1,1134 @@
+/*
+ * Copyright (C) 2016 by Open Source Routing.
+ *
+ * This file is part of GNU Zebra.
```

Comment:
Indeed. I'll take care of this in a separate commit (other files need to be updated as well).